core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -90,37 +90,46 @@ object ConsumerGroupCommand {
 
     protected def opts: ConsumerGroupCommandOptions
 
-    protected def getLogEndOffset(topic: String, partition: Int): LogEndOffsetResult
+    protected def getLogEndOffset(topic: Option[String], partition: Option[Int]): LogEndOffsetResult
```

Comment:
It seems weird to let this function accept `None` for either of these fields (why would I ever try to get the LEO if I don't have a topic or a partition?). Maybe the function can accept an instance of `TopicAndPartition` and the caller can make sure they have an instance prior to calling?